pkg/kn/commands/source/apiserver/describe.go

```diff
@@ -110,10 +109,10 @@ func writeResources(dw printers.PrefixWriter, apiVersionKindSelectors []v1alpha2
 
 func writeSink(dw printers.PrefixWriter, sink duckv1.Destination) {
 	subWriter := dw.WriteAttribute("Sink", "")
-	subWriter.WriteAttribute("Name", sink.Ref.Name)
-	subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
 	ref := sink.Ref
 	if ref != nil {
+		subWriter.WriteAttribute("Name", sink.Ref.Name)
+		subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
```

Comment:
I can comment out line 115 (and / or 116) and still pass all tests...

```go
func writeSink(dw printers.PrefixWriter, sink duckv1.Destination) {
	subWriter := dw.WriteAttribute("Sink", "")
	ref := sink.Ref
	if ref != nil {
		subWriter.WriteAttribute("Name", sink.Ref.Name)
		// subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
		// subWriter.WriteAttribute("Kind", fmt.Sprintf("%s (%s)", sink.Ref.Kind, sink.Ref.APIVersion))
	}
	uri := sink.URI
	if uri != nil {
		subWriter.WriteAttribute("URI", uri.String())
	}
}
```

Reply:
Sure, I'll need to update `createAPIServerSource`, which doesn't allow for setting a namespace currently. I don't think the original tests were covering this.

Reply:
Was able to just add `default` as namespace and kind as `Service (serving.knative.dev/v1)` to original test, which was only using `Service`. Doesn't cover since `ServiceAccount` is in the output.